src/Pipeline.cpp

```diff
@@ -582,6 +582,25 @@ void Pipeline::compile_jit(const Target &target_arg) {
     // Clear all cached info in case there is an error.
     contents->invalidate_cache();
 
+#ifdef WITH_SERIALZATION_JIT
+    // TODO: replace file serialization with in-memory serialization
```

Comment:
Nit: please open a Issue for this task and add the comment here e.g. `TODO(http://etc):`

Reply:
Derek already implement the API in his PR so I will refer to that